flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/ScheduledNotificationReceiver.java

```diff
@@ -26,14 +27,24 @@ public void onReceive(final Context context, Intent intent) {
         intent.getStringExtra(FlutterLocalNotificationsPlugin.NOTIFICATION_DETAILS);
     if (StringUtils.isNullOrEmpty(notificationDetailsJson)) {
       // This logic is needed for apps that used the plugin prior to 0.3.4
+
       Notification notification;
+      int notificationId = intent.getIntExtra("notification_id", 0);
+
       if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.TIRAMISU) {
         notification = intent.getParcelableExtra("notification", Notification.class);
       } else {
         notification = intent.getParcelableExtra("notification");
       }
+
+      if (notification == null) {
+        // This means the notification is corrupt
+        FlutterLocalNotificationsPlugin.removeNotificationFromCache(context, notificationId);
+        Log.e("notification", "Failed to parse a notification from  Intent. ID: " + notificationId);
```

Comment:
sorry i just missed that the tag here is normally the name of the file. something i should've caught for the other PR that dealt with exact alarm permissions. will change this one before it's released so thought i'd let you know

Reply:
Got it :) Thank you